Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -240,8 +239,10 @@ - (void)createSimulatorWithContext:(BPExecutionContext *)context {
         // If we failed to create the simulator, there's no reason for us to try to delete it, which can just cause more issues
         if (--__self.maxCreateTries > 0) {
```

Comment:
Are you saying to just remove this `if` and just always do what the `else` branch does in this case?